virus/Narnavirus_eurasiamusculus.md

```diff
@@ -0,0 +1,100 @@
+
+### N. eurasiamusculus is a narnavirus
+
+The SRA run was put into the Serratus.io [Edgar et al., 2022] RNA Dependent RNA Polymerase (RdRp) Search where the closest match to something that wasn't unclassified was 'Narnaviridae-7'. Subsequently, the palmprint sequence was then found in an associated virusRdRpCandidates.fasta document and traced back to the nucleotide sequence. From there, the palmprint amino acid sequence was inputted into blastp and the associated nucleotide sequence was inputted into blastx (both against the non-redundant protein sequences (nr) database) [Sayers et al., 2022]. Apart from one species of Riboviria, all other viruses that showed up in the results were narnaviruses. Thus, I concluded that this virus was a narnavirus.
+
+### N. eurasiamusculus only infects the skeletal and cardiac muscle tissues of goats and sheep
+
+Looking at all the SRA runs which had coverage for this virus, there were a few aspects that all or most runs had in common. The first was that all sequencing runs were done on samples that came from Capra hircus (goats) or Ovis aries (sheep). All BioProjects contained samples that originated from either Europe (Spain [1], Scotland [2]) or from Asia (India [1], Mongolia [1], Tibet [1], China [4]) (number in brackets denotes the number of bioprojects). After sorting the sequencing run metadata, there are a couple more patterns that stand out. Firstly, there is no bias in age or gender for which the virus is contained in the sample. Secondly, only with the exception of one sequencing run, all other samples for which the virus was found in was either in skeletal muscle or cardiac muscle. Important to note also that in all runs, there was no indication that the animals were sick or unhealthy prior to obtaining the sample. Additionally, many of the studies for which these SRA sequencing runs were associated with were for muscle development studies for meat quality. There is no significance in the virus being biased to one population over the other. For example, in the original study conducted in Gansu, the virus was found in all five Lioning cashmere goat samples. However, it was also found in 3 of the 5 Ziwuling black goat samples as well [Shen et al., 2021]. This is most likely due to contamination. In the study conducted in spain, this virus was only found in 2 out of 72 of the goat samples [Naldurtiker et al., 2022]. Thus, from the information we currenlty have, we can conclude that N. eurasiamusculus is a rare RNA virus that only infects the skeletal muscle and cardiac tissue of Eurasian goats and sheep.
+
+### Virus Genome
+
+![](images/figure.PNG)
```

Comment:
Figure is not included in the PR.

Please also add the really good "written" section associated with your Genome Map as the Figure Legend for this.

I would also move your "Figure 1" to the very start of your Results section to make this easier to read/interpret as a whole document.